src/components/templates/SimpleTable/SimpleTableManual.tsx

```diff
@@ -0,0 +1,249 @@
+      const theEditComponent = inRowField.type ? inRowField.type : 'not specified';
+      // For, display (readonly), the initial value (if there is one - otherwise, try placeholder)
+      //  and which component should be used for editing
+      return `${valBuilder !== '' ? valBuilder : thePlaceholder} (edit with ${theEditComponent})`;
+    }
+  }
+
+  // return the field from the incoming fields array that has "name" of
+  //  requested field
+  function getFieldFromFieldArray(inFieldName: string, inFieldArray: Array<any>): Object {
+    let objRet = {};
+
+    for (const field of inFieldArray) {
+      if (field?.config?.name === inFieldName) {
+        objRet = field;
+        break;
+      }
+    }
+
+    return objRet;
+  }
+
+  const formatRowsData = data => {
+    return data.map(item => {
+      return displayedColumns.reduce((dataForRow, colKey) => {
+        const theProcessedField = getFieldFromFieldArray(colKey, processedFields);
+        dataForRow[colKey] = getRowValue(item, colKey, theProcessedField);
+
+        return dataForRow;
+      }, {});
+    });
+  };
+
+  function generateRowsData() {
+    // if dataPageName property value exists then make a datapage fetch call and get the list of data.
+    if (dataPageName) {
+      getDataPage(dataPageName, context).then(listData => {
+        const data = formatRowsData(listData);
+        setRowData(data);
+      });
+    } else {
+      // The referenceList prop has the JSON data for each row to be displayed
+      //  in the table. So, iterate over referenceList to create the dataRows that
+      //  we're using as the table's dataSource
+      const data = formatRowsData(referenceList);
+      setRowData(data);
+    }
+  }
+
+  // These are the data structures referred to in the html file.
+  //  These are the relationships that make the table work
+  //  displayedColumns: key/value pairs where key is order of column and
+  //    value is the property shown in that column. Ex: 1: "FirstName"
+  //  processedFields: key/value pairs where each key is order of column
+  //    and each value is an object of more detailed information about that
+  //    column.
+  //  rowData: array of each row's key/value pairs. Inside each row,
+  //    each key is an entry in displayedColumns: ex: "FirstName": "Charles"
+  //    Ex: { 1: {config: {label: "First Name", readOnly: true: name: "FirstName"}}, type: "TextInput" }
+  //    The "type" indicates the type of component that should be used for editing (when editing is enabled)
+  //
+  //  Note that the "property" shown in the column ("FirstName" in the above examples) is what
+  //  ties the 3 data structures together.
+
+  // console.log("SimpleTable displayedColumns:");
+  // console.log(displayedColumns);
+  // console.log("SimpleTable processedFields:");
+  // console.log(processedFields);
+  // console.log(`SimpleTable rowData (${rowData.length} row(s)):`);
+  // console.log(JSON.stringify(rowData));
```

Comment:
I think these(and other console) statements can be removed.

Reply:
these are added by john, he actually added these as SimpleTable is not fully implemented. Let's keep these consoles, for now, we can delete them in the future.

Reply:
Yeah, just in case we need them for debugging, for now we can leave them in. Once the component is more mature, we can remove them.